manifests/runtime.yaml

```diff
@@ -21,3 +21,4 @@ spec:
     - name: app-proxy
       type: kustomize
       url: github.com/codefresh-io/cli-v2/manifests/app-proxy
+      wait: true
```

Comment:
will test several times the e2e, to see it consistently passes, before merging.
to validate we no longer have the race condition that we probably had before.